### DISCUSSION TITLE
Dúvida sobre angular (Aula Exibindo validações - Começando com Angular)

### DISCUSSION THREAD

**Discussion by Felipedacruzferreira** [balta-io/forum#291] [Tira Dúvidas] [answered]
Não reconhece o title
![image](https://user-images.githubusercontent.com/101299302/226940923-60db0d66-6e8f-44cf-aafc-e88d5328d2c5.png)
![image](https://user-images.githubusercontent.com/101299302/226941266-6c43f815-eb05-492d-b841-d92788be5a78.png)
![image](https://user-images.githubusercontent.com/101299302/226941357-f5629aeb-faa0-4f25-aa66-9924988df2fb.png)

não sei oq fazer , e nem estou sabendo como pesquisar meu problema

**Comment by andrebaltieri** [comment]
Bom dia, @Felipedacruzferreira poderia tentar `form.controls[0].title...`?

**Comment by Felipedacruzferreira** [comment]
Bom dia Balta!,o erro se manteve
![image](https://user-images.githubusercontent.com/101299302/227195154-55eb0a08-7d2e-4ce4-b7cd-4f37ff73aef3.png)

**Reply by Felipedacruzferreira** [reply]
![image](https://user-images.githubusercontent.com/101299302/227195437-a9597058-86f6-4a01-b5c6-8b45d0514bf8.png)

**Reply by andrebaltieri** [reply]
@Felipedacruzferreira falei errado, é `forms.controls['title'].invalid`... testa ai por favor

**Reply by MatheusVicentim** [reply]
Obrigado, isso me ajudou também.

**Comment by Felipedacruzferreira** [comment, marked as answer]
Oi balta, boa noite, ele ate resolveu só q agr deu outro problema
![image](https://user-images.githubusercontent.com/101299302/227373436-992fa531-c5b4-4813-86f9-120a7aa99b63.png)
![image](https://user-images.githubusercontent.com/101299302/227373487-75995a7e-9c39-4b6b-97de-d208c0b4c3d0.png)

**Reply by andrebaltieri** [reply]
Bom dia, @Felipedacruzferreira, qual problema?

**Reply by Felipedacruzferreira** [reply]
bom dia, o texto ao lado dos botões e a msg de  "alerta " ao lado da tarefa sumiram

**Reply by andrebaltieri** [reply]
Bom dia, @Felipedacruzferreira, no console não apareceu nenhum erro?

**Comment by john5ouza** [comment]
Fala [Felipedacruzferreira](https://github.com/Felipedacruzferreira), tenta assim, talvez resolva!

![code](https://user-images.githubusercontent.com/63263864/236697585-b927abf7-e0f6-445b-b5d6-884d93c3c940.png)

**Comment by lucccansousa** [comment]
![image](https://github.com/balta-io/forum/assets/131711446/1fd0e2ec-9244-4ec3-82fd-b7bd4fae251a)
não entendo esse problema   como json.parse (data)

**Reply by john5ouza** [reply]
No caso, tem que ter um IF em seguida para validação.
![code](https://github.com/balta-io/forum/assets/63263864/123ecbd2-404e-4b1e-ba99-1d78ddb90ae3)
